app/exporters/patients_exporter.rb

```diff
@@ -1,9 +1,8 @@
-require "csv"
-
 class PatientsExporter
+  require "csv"
   include QuarterHelper
 
-  BATCH_SIZE = 20
+  BATCH_SIZE = 500
```

Comment:
This performs a bit better because we're not creating that many throwaway objects anymore, so we can strain memory a bit to decrease overall time taken.